eea/dexterity/indicators/profiles/default/metadata.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <metadata>
-  <version>6.0</version>
+  <version>6.1</version>
```

Comment:
@dobri1408 No need to increase metadata version number as nothing changed in profiles / *.xml files.